cmd/levity/main.go

```diff
@@ -40,9 +41,12 @@ func init() {
 	flags.DurationVarP(&timeout, "timeout", "t", timeout,
 		"Timeout for GRPC requests")
 
-	flags.StringVar(&caCertPath, "ca", "", "Override CA file")
-	flags.BoolVar(&connectInsecure, "insecure", false,
-		"Connect insecurely to the server (i.e. no TLS)")
+	flags.StringVar(&caCertPath, "ca", "", "Override system CA with provided CA")
+
+	flags.StringVarP(&idPrivateKey, "key", "k", "./certs/usr-key.pem",
```

Comment:
This is different than the directory for the server, I think that expects certs to be in `./cert`. Also, this expects the certs to be in `bin` but the certs are placed in `repoRoot/certs`.

Reply:
I've removed the defaults for these options - they are now required arguments on CLI. I found the defaults less useful than I'd hoped, in any case.